base/expr.jl

```diff
@@ -118,6 +118,20 @@ macro eval(x)
     :($(esc(:eval))($(Expr(:quote,x))))
 end
 
+"""
+    @inline
+
+Small functions typically don't even need the @inline annotation,
+it does it on its own. By using @inline on bigger functions,
```

Comment:
Same as above with the single backticks around these two `@inline`s so that they're displayed correctly.